src-graphql/attribute-query.js

```diff
@@ -0,0 +1,5 @@
+import baseQuery from './base-query';
+
+export default function attributeQuery(fields = ['key', 'value']) {
```

Comment:
suggesting a rename to `lineItemAttributeQuery`, since `attributeQuery` is pretty ambiguous.